pkg/cluster/cluster.go

```diff
@@ -102,22 +102,25 @@ func new(config Config, s *spec.ClusterSpec, stopC <-chan struct{}, wg *sync.Wai
 		status:  &Status{},
 	}
 
-	if err := c.prepareBackupAndRestore(); err != nil {
-		if err == errBackupUnsetRestoreSet {
-			c.logger.Errorf("%v. This is not allowed. Cluster to be dead.", err)
+	if isNewCluster {
```

Comment:
I think the race is rare. Even under such race, we could guide that just delete the cluster and recreate it.

As a result, could we add a log to its "else" case that saying "restoring/recoving/refreshing states"?